### ISSUE TITLE
[BUG] Aruba box building instructions do not enable SSH server

### ISSUE THREAD

**Issue by ipspace** [ipspace/netlab#1117]
Using Aruba CX 10.13 box built according to https://netlab.tools/labs/arubacx/, the SSH server is not enabled in the management VRF and the box is never started (and could not be configured).

The missing command is **ssh server vrf mgmt**

@ssasso: I don't know enough about Aruba software to have an opinion about this. Is this needed in all ArubaCX versions, or is it a new thing? Should we add it to the initial device configuration?

**Comment by ssasso** [comment]
The command is present in the txt here:

https://github.com/ssasso/netlab/blob/9ccc0b43f5246552d2675e52673b5f8d75c40af9/netsim/install/libvirt/arubacx.txt#L16

Will give another checks in the next days

**Comment by ipspace** [comment]
> The command is present in the txt here:

My bad :( I should have looked.

However, we're facing another interesting challenge. I started building the box from scratch and:

* The **ssh server vrf mgmt** was in the initial configuration
* I applied the configuration recipe and the **ssh server vrf mgmt** command was in the config.
* I saved the configuration, reloaded the box, and the startup configuration contained **no ssh server vrf mgmt** as well as a shut-down mgmt interface.

I had to reapply the configuration, then it took.

Will rewrite the configuration recipe.
> 
> https://github.com/ssasso/netlab/blob/9ccc0b43f5246552d2675e52673b5f8d75c40af9/netsim/install/libvirt/arubacx.txt#L16
> 
> Will give another check in the next days
